tests/handler_tests/test_openai_handler.py

```diff
@@ -0,0 +1,26 @@
+import os
+import pytest
+
+from mindsdb.api.mysql.mysql_proxy.libs.constants.response_type import RESPONSE_TYPE
+from integration_tests.flows.http_test_helpers import HTTPHelperMixin
+from integration_tests.flows.conftest import *  # noqa: F403,F401
+
+# used by (required for) mindsdb_app fixture in conftest
+API_LIST = [
+    "http",
+]
+
+OPEN_AI_API_KEY = os.environ.get("OPEN_AI_API_KEY")
+
+
+@pytest.mark.usefixtures("mindsdb_app")
+class TestOpenAIHandler(HTTPHelperMixin):
```

Comment:
Should this file be added to CI in this PR or later?

Reply:
We can include them later